packages/textfield/src/input.tsx

```diff
@@ -131,7 +135,7 @@ export class TextFieldInput extends MaterialComponent<
             {...props}
           />
         )}
-        {props.label && this.state.jsComponentAttached && (
+        {props.label && this.state.jsComponentAttached && !outlined && (
```

Comment:
Shouln't the label be also there, when it is outlined?

Reply:
That's: https://github.com/prateekbh/preact-material-components/pull/1319/files#diff-8e917426ccc6e2794508642f1db54718R149

Reply:
Ah allright